Holy_locket.BLL/Holy_locket.BLL.csproj

```diff
@@ -7,6 +7,16 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="7.0.13" />
```

Comment:
BLL _shouldn't_ know details about DAL realization, you don't need to add these packages to BLL project